### PR TITLE
Translate getting-started into Japanese

### PR DESCRIPTION
**Pull request by oikwsat**
Dear AMP Project.

I've complete translate getting-started into Japanese.
(I copied getting-started.md and translated getting-started@ja.md)

Please take a look.

Thanks.

### CODE REVIEW

File: content/docs/getting-started@ja.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+$title: はじめに
+$order: 0
+toc: true
+---
+[TOC]
+
+これは、 AMP を始めるためのクイックスタートガイドです。
+
+詳細な説明については [初めての AMP ページを作成する](/docs/tutorials/create.html) チュートリアルをご覧ください。
```
Comment (bpaduch) [inline content/docs/getting-started@ja.md]: Point to the Japanese tutorial by prepending link with "/ja", like `/ja/docs/tutorials/create.html`

File: content/docs/getting-started@ja.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+{% call callout('もっと読む', type='read') %}
+AMP ページで [サポートされているCSS](/docs/guides/responsive/style_pages.html) をご覧ください。
+{% endcall %}
+
+### ステップ4: AMP HTML を検証する
+
+AMP ページが有効な AMP HTML であることを [AMP Validator](https://validator.ampproject.org/) で確認してください。
+
+使用できる他の検証ツールについては、 [AMP ページの検証](/docs/guides/validate.html) を見てください。
```
Comment (bpaduch) [inline content/docs/getting-started@ja.md]: Point to the Japanese doc by prepending link with: "/ja"

File: content/docs/getting-started@ja.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### 次のステップ
+
+更に AMP ページの基礎を知るためには、 [初めての AMP ページを作成する](/docs/tutorials/create.html) チュートリアルをご覧ください。
```
Comment (bpaduch) [inline content/docs/getting-started@ja.md]: Add `/ja` to URL

File: content/docs/getting-started@ja.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+あなたの役に立つ他のリソースがあります。
+
+* [ページを発見可能にする](/docs/guides/discovery.html)
```
Comment (bpaduch) [inline content/docs/getting-started@ja.md]: for the three links that are `/docs/...`, prepend links with `/ja`, so they are `/ja/docs...`.

**Comment by oikwsat** [comment]
@bpaduch Thanks for review!
I fixed review points and changed page titles.
Please take a look.

**Pull request merged**